__init__.py

```diff
@@ -137,6 +145,7 @@ def build_query(self, params):
 
     def request(self, data):
         """ Caching the responses """
+        # request is not used for local_api key
```

Comment:
Not necessary for this PR, but would be good for us to add caching support for local_api users in the future.

Reply:
should add a TODO here then